docs/content/Getting-Started/quickstart-subs/quickstart-1-install-and-run-kubestellar.md

```diff
@@ -1,9 +1,5 @@
 <!--quickstart-1-install-and-run-kubestellar-start-->
 
-KubeStellar works in the context of [kcp](https://www.kcp.io/), so to use KubeStellar you also need kcp.
-
-KubeStellar works with release `v0.11.0` of kcp.
```

Comment:
Don't we need to revise and extends these remarks, to talk about the space framework and a space provider?